src/analysis/normed/group/basic.lean

```diff
@@ -1136,6 +1136,27 @@ def normed_group.induced {E} [add_comm_group E]
 { .. semi_normed_group.induced f,
   .. metric_space.induced f h normed_group.to_metric_space, }
 
+/-- The constant norm that is `c` everywhere except at `dist x x = 0`.
+
+See note [reducible non-instances]-/
+@[reducible]
+def normed_group.const {E} [decidable_eq E] [add_comm_group E] (c : ℝ) (hc : 0 < c) :
+  normed_group E :=
```

Comment:
Since this is a def and not a global instance, I think it makes sense to leave that up to the caller. In discussion with @linesthatinterlace, the examples that came up certainly had decidable equality.